yarn-project/types/src/tx/tx_receipt.ts

```diff
@@ -40,6 +40,10 @@ export class TxReceipt {
      * The deployed contract's address.
      */
     public contractAddress?: AztecAddress,
+    /**
+     * Notes created in this tx.
+     */
+    public notes?: ExtendedNote[],
```

Comment:
Again, this is slightly misleading, since these are only the notes created in this tx that are visible by the current pxe.

Reply:
You are right. WDYT about calling it visibleNotes and documenting it properly? I think that should be sufficient because I would say the term "visible" will spark curiosity in devs.